dm-manager/cmd/dm-manager.go

```diff
@@ -99,21 +103,60 @@ func main() {
 		}
 	}
 
-	mpsClient, rpsClient, invTenantClient, eventsWatcher := prepareClients(transport)
+	mpsClient, err := mps.NewClientWithResponses(*mpsAddress, func(apiClient *mps.Client) error {
+		apiClient.Client = &http.Client{Transport: transport}
+		return nil
+	})
+	if err != nil {
+		log.Fatal().Err(err).Msgf("cannot create client")
+	}
+
+	rpsClient, err := rps.NewClientWithResponses(*rpsAddress, func(apiClient *rps.Client) error {
+		apiClient.Client = &http.Client{Transport: transport}
+		return nil
+	})
+	if err != nil {
+		log.Fatal().Err(err).Msgf("cannot create client")
+	}
+
+	orchMpsHost, orchMpsPort := getMpsAddress(infraConfigPath)
+
 	vsp := secrets.VaultSecretProvider{}
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
 	defer cancel()
 	if initErr := vsp.Init(ctx, []string{dm.AmtPasswordSecretName}); initErr != nil {
 		log.InfraSec().Fatal().Err(initErr).Msgf("Unable to initialize required secrets")
 	}
 
-	orchMpsHost, orchMpsPort := getMpsAddress(infraConfigPath)
+	dmReconciler := getDmController(mpsClient, rpsClient, vsp, orchMpsHost, orchMpsPort)
+	deviceReconciler := getDeviceController(mpsClient, rpsClient)
+
+	wg.Add(1)
+	go dmReconciler.Start()
+
+	wg.Add(1)
+	go deviceReconciler.Start()
+	<-osChan
+	termChan <- true
+
+	close(osChan)
+	close(termChan)
+	dmReconciler.Stop()
+	deviceReconciler.Stop()
+	wg.Wait()
+	log.Info().Msgf("Device Management Manager successfully stopped")
+}
 
+func getDmController(
```

Comment:
I am not sold on this name:
- either OpenDMTController
- better TenantController ?